source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -55,7 +55,8 @@ private string DetectUser()
         [CommonJobsAuthorize(Roles = "EmployeeManagers")]
         public ActionResult ReportDashboard()
         {
-            //TODO: Needs UI. Will call the API for the list (GetEvaluatedEmployees)
+            //Future: delete this line and bring the period dynamically
```

Comment:
I believe `//TODO` is a more standard indicator, and some IDEs will even recognize it.